apps/docs/src/guide/contracts/proxy-contracts.md

```diff
@@ -2,9 +2,51 @@
 
 Automatic deployment of proxy contracts can be enabled in `Forc.toml`.
 
-Once that is in place, [fuels deploy](https://docs.fuel.network/docs/fuels-ts/fuels-cli/commands/#fuels-deploy) will take care of it.
+We recommend that you use [fuels deploy](https://docs.fuel.network/docs/fuels-ts/fuels-cli/commands/#fuels-deploy) to deploy and upgrade your contract using a proxy as it will take care of everything for you. However, if you want to deploy a proxy contract manually, you can follow the guide below.
 
-## Docs
+## Manually Deploying and Upgrading by Proxy
+
+As mentioned above, we recommend using [fuels deploy](https://docs.fuel.network/docs/fuels-ts/fuels-cli/commands/#fuels-deploy) to deploy and upgrade your contract as everything is handled under the hood. But the below guide will detail this process should you want to implement it yourself.
+
+We recommend using the [SRC14 compliant owned proxy contract](https://github.com/FuelLabs/sway-standard-implementations/tree/master/src14/owned_proxy) as the underlying proxy and that is the one we will use in this guide, as is the one used by [fuels deploy](https://docs.fuel.network/docs/fuels-ts/fuels-cli/commands/#fuels-deploy).
```

Comment:
Should we link to a specific commit hash instead of `master`?

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3253/commits/8f9c67c66cf47536f6fd43fa7952d97b7da2ecd2